pulp_ansible/app/tasks/collections.py

```diff
@@ -64,7 +65,43 @@
 log = logging.getLogger(__name__)
 
 
-def sync(remote_pk, repository_pk, mirror):
+def update_collection_remote(remote_pk, *args, **kwargs):
+    """
+    Update a CollectionRemote.
```

Comment:
Can this docstring get small paragraph after that describes the other behaviors it does besides updating a CollectionRemote?